fab/config.example.py

```diff
@@ -0,0 +1 @@
+GITHUB_APIKEY = ''
```

Comment:
Would be nice to add a comment here with minimal steps to perform on github to get an API key.